src/termwidgetholder.cpp

```diff
@@ -390,6 +390,7 @@ void TermWidgetHolder::setCurrentTerminal(TermWidget* term)
         {
             emit termTitleChanged(windowTitle(), QString{});
         }
+        emit termFocusChanged();
```

Comment:
Could you use `Q_EMIT` instead? In the long term, I'd like to introduce `QT_NO_KEYWORDS` into qterminal/qtermwidget. Feel free to replace other `emit` usage you see nearby :)

Reply:
Sure.